TestAssets/TestPackages/dotnet-dependency-tool-invoker/project.json

```diff
@@ -3,19 +3,18 @@
   "compilationOptions": {
     "emitEntryPoint": true
   },
+  "compile": [
+    "**/*.cs",
+    "../../../src/dotnet/CommandLine/*.cs"
```

Comment:
Because then you need to do InternalsVisibleTo.  And this tool would need to load dotnet.dll in order to run.  This is technically what it was doing before - compiling the same code in both assemblies.